### ISSUE TITLE
mips decompiler

### ISSUE THREAD

**Issue by spein46** [NationalSecurityAgency/ghidra#1296]
Hi, in disassembler solve referece to function pointer good, but decompiler doesn't solve, indeed I get behavior weird
A clear and concise description of the bug.

Expected behavior
I expext decompiler solve reference like disassembler

**Screenshots**
<img width="679" alt="Screenshot 2019-11-28 at 19 00 39" src="https://user-images.githubusercontent.com/17325529/69825957-67984d80-1211-11ea-9bd6-345950c7f3b9.png">
<img width="896" alt="Screenshot 2019-11-28 at 19 00 23" src="https://user-images.githubusercontent.com/17325529/69825959-67984d80-1211-11ea-9d91-a824ada95185.png">

**Environment (please complete the following information):**
 - OS: [e.g. macOS 10.15.1]
 - Java Version: [e.g. 12.0.1]
 - Ghidra Version: [e.g. 9.2]

**Comment by paulsapps** [comment]
seem to be a duplicate of #1289

**Comment by spein46** [comment]
yes, sorry, I think this is better than "non-compliant decompiler", do you have solution?
